package.json

```diff
@@ -81,7 +81,7 @@
     "@artsy/express-reloadable": "1.4.8",
     "@artsy/fresnel": "1.3.1",
     "@artsy/gemup": "0.1.0",
-    "@artsy/palette": "14.18.0",
+    "@artsy/palette": "14.18.3",
```

Comment:
Interestingly there's still a bug here, notice: https://www.artsy.net/collect?page=100 this page still `hasNextPage` according to MP/Relay. We could potentially work around this in Palette, but it looks like the bug lies elsewhere. Investigating.